dali/pipeline/operator/arg_helper.h

```diff
@@ -183,6 +269,17 @@ class ArgValue {
     return view_[idx];
   }
 
+  /**
+   * @brief true if the argument is empty or the particular sample
+   *        has a 0-volume shape
+   */
+  bool IsEmpty(size_t idx) const {
```

Comment:
```suggestion
  bool IsEmpty(int idx) const {
```
We usually use `int` as sample index.